### PR TITLE
updated few parts of readme, setting up the environment left issue #22 

### PR DESCRIPTION
**Pull request by Dragonstac**
Cleaned up the readme, only setting up the environment is left and seperated other files into docs foldes and linked them to the main README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,537 +2,25 @@
...
-         .assert.textEquals('.pcs-edit-section-title', 'BrowserStack');  // command run in webview context
-     });
-   });
-   ```
-6. (**Real Device**) Run the following command to get the *UDID*:
-   ```sh
-    system_profiler SPUSBDataType | sed -n '/iPhone/,/Serial/p' | grep 'Serial Number:' | awk -F ': ' '{print $2}'
-    ```
+- [Android](docs/ANDROID.md)
+- [IOS](docs/IOS.md)
```
Comment (garg3133) [inline README.md:17]: Keep the format for this to be the same as before. Also, we should bifurcate the usage docs based on Mobile web testing and Mobile app testing, and not Android and iOS. Sorry if I mentioned dividing the docs for Android and iOS before.

File: README.md [hunk 2]
```diff
@@ -2,537 +2,25 @@
...
 
-7. (**Optional**) Update the configurations : 
+## Contributing
 
-    **Real Device**
+We welcome any and all contributions from the community which can help improve Nightwatch. Please check out [CONTRIBUTING.md](docs/CONTRIBUTING.md) for more extensive contributing guidelines.
```
Comment (garg3133) [inline README.md:21]: The contributing guide linked here has many things specific to Nightwatch. Maybe we can just remove the contributing guide for now and list the steps to setup the project and contribute to it briefly in the README itself.

File: README.md [hunk 3]
```diff
@@ -2,537 +2,25 @@
...
-         .waitUntil(async function() {
-           // wait for webview context to be available
-           const contexts = await this.appium.getContexts();
-
-           return contexts.length > 1;
-         }, 5000)
-         .perform(async function() {
-           // switch to webview context
-           const contexts = await this.appium.getContexts();
+Nightwatch helper-tool can be used on both Android and IOS devices. You can use the guides below to use it on your devices.
```
Comment (garg3133) [inline README.md:14]:
```suggestion
This tool can be used to set up mobile-web and mobile-app testing on both Android and iOS devices. Please follow the usage guides below to set up mobile-testing in your existing Nightwatch.js project.

To set up a new Nightwatch.js project with support for mobile-testing out-of-the-box, run:
```sh
npm init nightwatch@latest <project-name>
```[x]
and answer the questions that follow.

```

**Review by garg3133 — COMMENTED** [review]
@Dragonstac Always create a new branch whenever making any changes. You should never make changes directly to the `main` branch.

`CODE_OF_CONDUCT.md`, `CONTRIBUTING.md` and `LICENSE.md` should be present in the root folder itself.

File: README.md (hunk 2 above)
Comment (Dragonstac) [reply README.md:21]: ok

File: README.md (hunk 1 above)
Comment (Dragonstac) [reply README.md:17]: Ok will bifurcate into web and android testing

**Pull request closed without merging**